### DISCUSSION TITLE
Reference code in nonlinear elasticity and large strains

### DISCUSSION THREAD

**Discussion by rebpoli** [libMesh/libmesh#3989] [General]
Hi

I am looking for a libmesh reference code which solves nonlinear elasticity within the large strains framework.
Is there any example to suggest?

Thank you,
Renato

**Comment by dknez** [comment]
There is a simple large strain elasticity example here:
https://github.com/libMesh/libmesh/tree/devel/examples/systems_of_equations/systems_of_equations_ex7

David
